tests/e2e/test_rosacli_kubelet_config.go

```diff
@@ -33,9 +33,9 @@ var _ = Describe("Kubeletconfig on Classic cluster",
 			kubeletService = rosaClient.KubeletConfig
 
 			By("Skip testing if the cluster is a HCP cluster")
-			isClassic, err := rosaClient.Cluster.IsHostedCPCluster(clusterID)
+			isHosted, err := rosaClient.Cluster.IsHostedCPCluster(clusterID)
 			Expect(err).ToNot(HaveOccurred())
-			if !isClassic {
+			if isHosted {
 				SkipNotClassic()
```

Comment:
From my side, I used `!isCLassic` to match `SkipNotClassic()` to improve the readability. If you think the current logic is better, please also update `test_rosacli_machine_pool.go` which defines `isClassic` too,
```
			if isHosted {
				SkipNotClassic()

Reply:
`test_rosacli_machine_pool.go` has already been changed in https://github.com/openshift/rosa/pull/2135, This is a leftover